packages/api/src/templates/locales/fr.json

```diff
@@ -90,12 +90,12 @@
     "MANAGER": "Gestionnaire"
   },
   "EXPORT": {
-    "HI": "EXPORT.HI",
-    "SEASON": "EXPORT.SAISON",
-    "CERTIFICATION": "EXPORT.CERTIFICATION",
-    "GET_YOUR_EXPORT": "EXPORT.OBTENEZ_VOTRE_EXPORTATION",
-    "SUBJECT": "EXPORT.SUBJET",
-    "PRIVACY": "EXPORT.CONFIDENTIALITÉ",
-    "LANGUAGE_DELAY": "EXPORT.DÉLAIS.LANGUE"
+    "HI": "Bonjour",
+    "SEASON": "Nous espérons que vous avez eu une saison fantastique à",
+    "CERTIFICATION": " Vous pouvez cliquer sur le lien ci-dessous pour télécharger un fichier zip de vos documents de certification et tous les documents à l'appui (photos d'emballages de semences, reçus d'entrée, etc.) pour votre période de déclaration.",
+    "GET_YOUR_EXPORT": "Obtenir vos documents",
+    "SUBJECT": "Export de documents de certification de LiteFarm",
+    "PRIVACY": "Chez LiteFarm, nous prenons votre vie privée au sérieux, donc vous devrez être connecté avant que vous puissiez accéder votre export. Désolée pour le dérangement. Bonne chance avec votre soumission et merci d'utiliser LiteFarm!",
+    "LANGUAGE_DELAY": ""
```

Comment:
Is this empty string value intentional?

Reply:
Good catch! I totally forgot to translate that one. It should be fixed now.